src/components/sidenav/styles/sidebar-desktop.scss

```diff
@@ -93,22 +93,24 @@
   border-radius: 6px;
   height: 38px;
   width: 176px;
+  margin-left: 24px;
 }
 
+// adding -120px to the margin-top to adjust the position because of the lgfm logo
 .menu__assets {
-  margin-top: -276px;
+  margin-top: calc(-276px - 120px);
 }
 
 .menu__dashboard {
-  margin-top: -230px;
+  margin-top: calc(-230px - 120px);
 }
 
 .menu__staking {
-  margin-top: -184px;
+  margin-top: calc(-184px - 120px);
 }
 
 .menu__bridge {
-  margin-top: -138px;
+  margin-top: calc(-138px - 120px);
```

Comment:
Same as above, why the `calc()`?

Reply:
because there is a possibility to remove this banner in future, I thought that it's better to use calc to not calculate back again and prevent mis-calculation the values

Reply:
Since it's for documentation purpose, I'd advise using a comment instead of calc.